### PR TITLE
styling all colors using bootstrap's classes

### PR DESCRIPTION
**Pull request by yatharthMogra**
Added "bg-primary"  to class list of chat-msg

Screenshot from 2020-10-08 12-16-58

### CODE REVIEW

**Comment by LalitNM** [comment]
One thing which you need to consider is a little spike in message bubble. And there are other regions in app where color is not styled by BS's classes, like the plus button at the top and the message place. You need to change these sections also.

**Review by yatharthMogra — COMMENTED** [review]
=>replacedd the background color with that of "bg-primary" of
 two elements in blue.
=> removed the spike contrast

Screenshot from 2020-10-08 15-35-32

**Review by yatharthMogra — COMMENTED** [review]
=> background color change for plus-circle icon

Screenshot from 2020-10-08 15-49-51

**Comment by yatharthMogra** [comment]
@LalitNM please review the changes

File: public/style.css [hunk 1]
```diff
@@ -228,7 +228,7 @@ body {
   margin-left: -1px;
   padding-top: 13px;
   border: 0;
-  background-color: #00b0ff;
+  background-color: #007bff;
```
Comment (LalitNM) [inline public/style.css:230]: Remove this color from here and add `bg-primary` class to respective places in code.
And do same at all places in code.

**Review by LalitNM — CHANGES_REQUESTED** [review]

**Review by yatharthMogra — COMMENTED** [review]
=>at 2 of 4 places i have added "bg-primary" class
=>at the msg-bubble_spike and element person:active adding a class does not seems viable hence i have used css property to render them.

Screenshot from 2020-10-08 15-35-32

**Review by yatharthMogra — COMMENTED** [review]
=> changed color of send icon by using "text-primary" class

Screenshot from 2020-10-08 16-31-18

**Review by yatharthMogra — COMMENTED** [review]
=> changed the appearence of join room and also centered it so as to avoid overlaying of content
=> added jumbotron class to error (the grey background behind join room)

Screenshot from 2020-10-08 17-55-22

=> changed the border color by BS4 in users.ejs
=>removed irrerelevant css

File: public/script.js [hunk 2]
```diff
@@ -62,7 +62,7 @@ function showRoom(name) {
   $("#" + room_id + "-msg").css("display", "inherit").addClass('active');
   if ($("#" + room_id + "-msg").attr("data-joined") == 0) {
     $(".error").css("display", "inherit");
-    $(".error").html('<span id="error">You havent joined this room yet. <input type="button" onclick="joinRoom(\'' + name.id + '\')" value="Join" id="joinBtn"/> to see the conversation.</span>');
+    $(".error").html('<span id="error">You havent joined this room yet. <a onclick="joinRoom(\'' + name.id + '\')" id="joinBtn">Join to see the conversation.</span>');
```
Comment (LalitNM) [inline public/script.js]: this `<a>` tag is not closing, unlike `<input>` tags `<a>` tag has a closing. This makes all lines after 'join' a clickable area. And also try to use something like
```
const el = document.createElement('span');
el.setAttribute("id", "error");
```
It is always a good practice to write clean code.

File: public/style.css [hunk 3]
```diff
@@ -435,11 +434,7 @@ body {
 
 #joinBtn {
   cursor: pointer;
-  background: #1ade1a;
-  color: white;
   width: 20%;
-  padding: 1%;
-  border: 0;
 }
```
Comment (LalitNM) [inline public/style.css]: This(whole styling of `#joinBtn`) can be removed I guess:thinking:

File: public/client.js [hunk 4]
```diff
@@ -43,7 +43,7 @@ function leaveRoom(room) {
     socket.emit('leave room', {
         name: room
     });
-    $(".error").html('<span id="error">You havent joined this room yet. <button onclick="joinRoom( \'' + room + '\' )" id="joinBtn">Join<Button/> to see the conversation.</span>');
+    $(".error").html('<span id="error">You havent joined this room yet. <a onclick="joinRoom( \'' + room + '\' )" id="joinBtn">Join to see the conversation.</a></span>');
```
Comment (LalitNM) [inline public/client.js]: Ty to use something like
```
const el = document.createElement('span');
el.setAttribute("id", "error");
```
It is always a good practice to write clean code.

**Review by LalitNM — CHANGES_REQUESTED** [review]

**Comment by LalitNM** [comment]
One more thing you have to take care is that the jumbotron is present even when we are in a room for reference
image

**Review by yatharthMogra — COMMENTED** [review]
=> added missing closing a tag to public/script.js
=> Removing joinBtn class from css is not viable as user needs to know where to click upon
 => !!Issue!! appearence of div.error when site is opened for first time

**Review by yatharthMogra — COMMENTED** [review]
=>Only probable option for resolving display issue seems to be setting it's default display to none

**Review by yatharthMogra — COMMENTED** [review]
Anchor tag with href=#

**Review by LalitNM — APPROVED** [review]

**Comment by LalitNM** [comment]
Merged :rocket: good work @yatharthMogra.

**Pull request merged**